orders/templates/orders/overview.html

```diff
@@ -51,7 +51,7 @@ <h2>Bestellingen ({{ grandtotal | euro }})</h2>
                     </p>
                     <p>
                         Als je graag op het randje leeft kan dat ook met:<br>
-                        <span style='font-size:12px;font-family:Menlo,Monaco,Consolas,"Courier New",monospace;'>curl https://joostrijneveld.nl/eetfestijn/print/ | sh</span>
+                        <span style='font-size:12px;font-family:Menlo,Monaco,Consolas,"Courier New",monospace;'>curl {{ print_url }} | sh</span>
```

Comment:
Instead of using a var from the view, just use the `url` templatetag.

Reply:
`url 'print_script'` generates a relative url